go-controller/pkg/libovsdbops/mac_binding.go

```diff
@@ -9,10 +9,15 @@ import (
 // CreateOrUpdateMacBinding creates or updates the provided mac binding
 func CreateOrUpdateMacBinding(sbClient libovsdbclient.Client, mb *sbdb.MACBinding) error {
 	opModel := operationModel{
-		Model:          mb,
-		OnModelUpdates: onModelUpdatesAll(),
-		ErrNotFound:    false,
-		BulkOp:         false,
+		Model: mb,
+		OnModelUpdates: []interface{}{
+			&mb.Datapath,
+			&mb.LogicalPort,
+			&mb.IP,
+			&mb.MAC,
```

Comment:
Better to have a fields argument and let them be provided by the caller, as we do in some other places?

Reply:
done